v2/common/src/main/java/com/google/cloud/teleport/v2/transforms/ConstructGenericRecordsFn.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.transforms;
+
+import java.util.Collections;
+import java.util.Map;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.GenericRecordBuilder;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.values.KV;
+
+/**
+ * The {@link ConstructGenericRecordsFn} class that converts KafkaMessages to Generic Records to
+ * write it to Avro or Parquet File Format.
+ */
```

Comment:
This class converts any KV<String, String> to a GenericRecord with a static schema. Can we change the description to say that somehow. I dont think this has anything to do with KafkaMessages.